__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => seed(data));
+
+describe("app", () => {
+  describe("GET /api/topics", () => {
+    test("Status: 200, responds with an array of topic objects which include the slug and description properties", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body: { topics } }) => {
+          topics.forEach((topic) => {
+            expect(topic).toEqual(
+              expect.objectContaining({
+                description: expect.any(String),
+                slug: expect.any(String),
+              })
+            );
+          });
+        });
+    });
```

Comment:
Make sure that you are expecting how many topics are coming back. It could be an empty array